### ISSUE TITLE
Should the target of aria-errormessage in UIA have a flag in AriaProperties?

### ISSUE THREAD

**Issue by sivakusayan** [w3c/core-aam#183]
Currently `aria-errormessage` is exposed in UIA [via the ControllerFor property](https://www.w3.org/TR/core-aam-1.2/#ariaErrorMessage). This means that, as far as I can tell, an API consumer won't be able to distinguish between targets of `aria-errormessage` and targets of `aria-controls` without hacky workarounds, since `aria-controls` [also uses ControllerFor](https://www.w3.org/TR/core-aam-1.2/#ariaControls). 

Should the target of an `aria-errormessage` be given some property in `AriaProperties` to distinguish it from being the target of an `aria-controls`? Or could this information possibly be exposed in a different UIA property?

**Comment by douggeoffray** [comment]
Using ControllerFor doesn't make sense to me. Not just because it conflicts with aria-controls, but because the element isn't really controlling the element containing the error. Yes, it caused it to generate but not really controlling it beyond its presence. To me, the error is a form of a description. This brings up the UIA FullDescription property and the DescribedBy property. I don't like FullDescription as that would conflict with valid data as well. Given DescribedBy is an array of elements, this makes the most sense to me.

How does this sound? We are having internal discussions on this as I really don't think ControllerFor is the right property.

**Comment by sivakusayan** [comment]
I agree that the ControllerFor mapping was confusing, but I wasn't sure if there was a reason for it. Aligning it with the UIA description makes more sense to me, too.
